railties/test/application/rake/dbs_test.rb

```diff
@@ -86,6 +86,25 @@ def db_create_and_drop(expected_database, environment_loaded: true)
         db_create_and_drop("db/development.sqlite3", environment_loaded: false)
       end
 
+      test "db:create and db:drop show warning but doesn't raise errors when loading YAML with alias ERB" do
+        app_file "config/database.yml", <<-YAML
+          sqlite: &sqlite
+            adapter: sqlite3
+            database: db/development.sqlite3
+
+          development:
+            <<: *sqlite
```

Comment:
Should not this be an ERB? This seems to be a yaml that we can read without problem.

Reply:
Oops yes, will fix that.

Reply:
Actually this is straight up a bad test 😢 Working on fixing it.